docs/core/tools/dotnet-environment-variables.md

```diff
@@ -183,6 +183,18 @@ See [EventPipe environment variables](../diagnostics/eventpipe.md#trace-using-en
 
 Specifies the location of the .NET runtimes, if they are not installed in the default location. The default location on Windows is `C:\Program Files\dotnet`. The default location on Linux and macOS is `/usr/local/share/dotnet`. This environment variable is used only when running apps via generated executables (apphosts). `DOTNET_ROOT(x86)` is used instead when running a 32-bit executable on a 64-bit OS.
 
+### `DOTNET_HOST_PATH`
+
+Specifies the absolute path to a `dotnet` host (`dotnet.exe` on Windows, `dotnet` on non-Windows) that was used to launch the currently-running `dotnet` process. This is used by the .NET SDK to help tools that run during .NET SDK commands ensure they use the same `dotnet` runtime for any child `dotnet` processes they create for the duration of the command. Tools and MSBuild Tasks within the SDK that invoke binaries via the `dotnet` host are expected to honor this environment variable to ensure a consistent experience.
```

Comment:
"non-Windows" - Can you use another term than that? It gives me the feels of an older Microsoft. How about (and I'm going on a limb here) "on Linux and macOS".

Reply:
sure - I was going for maximally-inclusive terms here but I can see your point.

Reply:
```suggestion
Specifies the absolute path to a `dotnet` host (`dotnet.exe` on Windows, `dotnet` on Linux and macOS) that was used to launch the currently-running `dotnet` process. This is used by the .NET SDK to help tools that run during .NET SDK commands ensure they use the same `dotnet` runtime for any child `dotnet` processes they create for the duration of the command. Tools and MSBuild Tasks within the SDK that invoke binaries via the `dotnet` host are expected to honor this environment variable to ensure a consistent experience.
```

Reply:
```suggestion
Specifies the absolute path to a `dotnet` host (`dotnet.exe` on Windows, `dotnet` on Linux and macOS) that was used to launch the currently-running `dotnet` process. This is used by the .NET SDK to  ensure that tools (and any processes they create) that run as part of .NET SDK commands use the same `dotnet`. Tools and MSBuild Tasks within the SDK that invoke .NET apps via the `dotnet` host are expected to honor this environment variable to ensure a consistent experience.
```